ITP-May | Onboarding 1 | Roja Alagurajan | Form-Controls

alagurajan-roja opened the pull request:
<!--

You must title your PR like this:

REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME

For example,

London | May-2025 | Carol Owen | Wireframe

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [ ] I have committed my files one by one, on purpose, and for a reason
- [ ] I have titled my PR with REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME
- [ ] I have tested my changes
- [ ] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [ ] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

cjyuan reviewed (commented):
Code is error free and well indented. Form looks good!

1. Can you enforce a stricter validation rule for full name?

2. I saw some useful suggestions  when I asked ChatGPT this:
```
review HTML code:
...        <--- paste your HTML code here
```

In particular, it pointed out some inconsistency in the code.

To get better at using AI tools, can you try running your code by ChatGPT? Take a look at the suggestions it gives you, and pick out the ones that actually make sense for your project. It's a great way to learn and improve your work.

alagurajan-roja commented:
@cjyuan 
i have corrected my code. Can you please review again

cjyuan commented on Form-Controls/index.html:
```diff
@@ -1,27 +1,55 @@
+       <!-- T-Shirt Colour -->
+      <fieldset>
+        <legend>Select T-Shirt Colour:</legend>
+        <label><input type="radio" name="colour" value="green" required> Green</label><br>
+        <label><input type="radio" name="colour" value="blue"> Blue</label><br>
+        <label><input type="radio" name="colour" value="Yellow" >Yellow</label>
```
There is some inconsistency in terms of spacing and letter case among these three radio buttons.

Also, for accessibility, each <label> should use the `for` attribute pointing to the corresponding input id.
Can you ask ChatGPT to review your code and see what it suggests?

alagurajan-roja commented:
Hi @cjyuan ,

Thank you for your feedback! I have carefully reviewed your suggestions and made the necessary changes with the help of ChatGPT.

Specifically:

I made changes to the title attribute for better clarity.

I updated the radio buttons to use <fieldset> and <legend> for improved accessibility and structure.

I ensured each <input> has a unique id and each <label> uses the corresponding for attribute.

I wrapped each radio button and label inside <div> elements to improve layout and styling flexibility.

I have replaced the “Reviewed” label with “Needs review” so you can kindly check the updates again.

Please let me know if anything else needs to be changed in my code.

Thanks again!

Best regards,
Roja Alagurajan

cjyuan commented:
Changes look good, but now the code is not properly indented.

cjyuan commented:
Indentation looks better now. Why not just use VSCode "Format Document" feature to indent the code?
